internal/app/api_processor.go

```diff
@@ -122,7 +122,18 @@ func syncResources(resources []string, source *conjurapi.Client, destination *co
 	return
 }
 
-func addSecret(destination *conjurapi.Client, variable string, value string) error {
+func addSecret(destination *conjurapi.Client, variable string, value string, skipSameValue bool) error {
+	// Skip writing values that are the same in both systems
+	if skipSameValue {
+		// Ignore errors, because it could indicate a missing value which should be set
+		destinationSecret, _ := destination.RetrieveSecret(variable)
+
+		if string(destinationSecret) == value {
+			fmt.Printf("Skipping variable: %s\n", variable)
```

Comment:
```suggestion
			fmt.Printf("Variable matches, not updating: %s\n", variable)
```
Slight change in comment to show that the variable isn't ignored, its value has been checked.